src/ethereum/frontier/trie.py

```diff
@@ -49,13 +49,16 @@
 #   1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347 # noqa: E501,SC10
 #
 # which is the sha3Uncles hash in block header with no uncles
-EMPTY_TRIE_ROOT = hex_to_bytes(
-    "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
+EMPTY_TRIE_ROOT = Root(
+    hex_to_bytes(
+        "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
+    )
 )
 
 Node = Union[Account, Bytes, Transaction, Receipt, Uint, U256, None]
-T = TypeVar(
-    "T",
+K = TypeVar("K", bound=Bytes)
```

Comment:
Can we avoid single-letter names like these? Can we call this `TrieKey` instead?

Reply:
Ah, I didn't realize you were talking about generic arguments on the sync call. I fairly strongly prefer single letter generic arguments.

Reply:
Okay, I don't have a strong opinion in that case. Feel free to merge.